libs/Helpers.py

```diff
@@ -96,3 +96,14 @@ def convert_to_hours_and_minutes(self, seconds):
         m, s = divmod(int(seconds), 60)
         h, m = divmod(m, 60)
         return h, m
+
+    @keyword
+    def install_isv_by_name(self, operator_name, channel, source="certified-operators"):
+        ocm_client = OpenshiftClusterManager()
+        ocm_client.install_openshift_isv(operator_name, channel, source, exit_on_failure=False)
+        if operator_name == "ovms":
+            status = ocm_client.wait_for_isv_installation_to_complete("openvino")
```

Comment:
why this custom check on openvino?

Reply:
When we try to check if operator is installed or not using get csv. For open vino it doesn't consisting the name ovms for openvino operator unlike aikit and pachyderm.

Reply:
Agree. I have the same idea that why i created dictionary. My plan was to watch out for other ISV and then modify it .